docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
-   If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
+    .. tab:: Pypi Installation
 
-   ::
+        Install ``ivadomed`` and its requirements from
+        `Pypi <https://pypi.org/project/ivadomed/>`__:
 
-     virtualenv venv-ivadomed --python=python3.6
-
-Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ::
 
+            pip install --upgrade pip
 
-::
+            pip install ivadomed
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+    .. tab:: Repo Installation (Advanced or Developer)
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        Bleeding-edge developments are available on the project's master branch
+        on Github. Installation ``ivadomed`` from source:
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+        ::
 
-::
-   
-   pip install -r requirements_gpu.txt
+            git clone https://github.com/ivadomed/ivadomed.git
 
+            cd ivadomed
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
+            pip install -e .
 
 
-::
+Step 3: Install ``ivadomed`` with CPU or GPU Support, Install ``torch`` and ``torchvision``
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    pip install -r requirements.txt
+.. tabs::
 
+    .. tab:: CPU Support
 
-Step 4: Install from release (recommended)
-++++++++++++++++++++++++++++++++++++++++++
+        If you plan to run ``ivadomed`` on CPU only, run the following command:
+        ::
 
-Install ``ivadomed`` and its requirements from
-`Pypi <https://pypi.org/project/ivadomed/>`__:
+            pip install -r requirements.txt
 
-::
 
-    pip install --upgrade pip
-    pip install ivadomed
+    .. tab:: Nvidia GPU Support
 
-(Optional) Alternative Step 4 for Developers: Install from source
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-Bleeding-edge developments are available on the project's master branch
-on Github. Installation procedure is the following at repository root:
+        If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed, try run the following command:
 
-::
+        ::
 
-    cd ivadomed
-    pip install -e .
+           pip install -r requirements_gpu.txt
 
+        Please note that this must happens after the previous ``ivadomed`` installation step.
```

Comment:
I think that this line might be a little unnecessary, just because I think it's a fair expectation that people should follow steps in order, hehe.

Reply:
Good point. I was actually thinking about this too.